sample-spring-boot/README.md

```diff
@@ -8,8 +8,9 @@ See section [Features](#features) for details.
 ## Getting Started
 
 This sample application is contained in the Maven module sample-spring-boot.
-It can be built with a simple `mvn install`. 
 A minimal CF application manifest is provided in [manifest.yml.](manifest.yml)
+You should change the name before building the app to avoid naming conflicts.
```

Comment:
The documentation is written in passive without addressing the reader.

Reply:
pushed fix